2018-06-07 03:23:47,087: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:23:47,088: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:23:48,353: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:23:48,353: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:23:49,679: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:23:49,679: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:23:50,764: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:23:50,764: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:24:00,951: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:24:00,951: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:25:21,143: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:25:21,143: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got an unexpected keyword argument 'path'
2018-06-07 03:25:22,825: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:25:22,826: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got an unexpected keyword argument 'path'
2018-06-07 03:26:40,057: Not Found: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
2018-06-07 03:26:41,735: Not Found: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
2018-06-07 03:27:02,143: Not Found: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
2018-06-07 03:27:21,018: Not Found: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
2018-06-07 03:27:22,720: Not Found: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
2018-06-07 03:27:42,598: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:27:42,598: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:28:25,673: Not Found: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
2018-06-07 03:28:52,614: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:28:52,615: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:28:54,075: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:28:54,075: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:31:33,217: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:31:33,223: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:33:14,879: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:33:14,879: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:33:16,411: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:33:16,411: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:33:17,864: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:33:17,864: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:33:19,786: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:33:19,786: Internal Server Error: /imageview/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: imageview() got multiple values for argument 'path'
2018-06-07 03:34:13,163: tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:34:33,313: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:34:33,314: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:34:33,314: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:34:33,314: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:34:33,314: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:34:33,314: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:34:33,330: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:35:44,107: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:35:44,108: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:35:44,108: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:35:44,108: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:35:44,108: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:35:44,108: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:35:44,121: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:38:28,297: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:38:28,306: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:38:28,306: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:38:28,307: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:38:28,307: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:38:28,307: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:38:28,323: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:38:28,324: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:38:28,325: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:38:28,326: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:38:54,145: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:38:54,146: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:38:54,146: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:38:54,146: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:38:54,146: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:38:54,146: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:41:18,517: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:41:18,517: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:41:18,518: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:41:18,518: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:41:18,518: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:41:18,518: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:41:18,540: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:41:18,541: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:41:18,543: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:41:18,545: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:41:18,547: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:41:18,548: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:42:06,055: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:42:06,056: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:42:06,056: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:42:06,056: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:42:06,056: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:42:06,056: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:42:06,075: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:42:06,077: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:42:06,079: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:42:06,082: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:42:06,084: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:42:06,087: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:42:51,268: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:42:51,269: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:42:51,269: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:42:51,269: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:42:51,269: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:42:51,270: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:42:51,292: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:42:51,294: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:42:51,296: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:42:51,298: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:42:51,300: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:42:51,302: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:43:37,901: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:43:37,901: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:43:37,902: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:43:37,902: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:43:37,902: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:43:37,902: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:43:37,917: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:43:37,919: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:43:37,920: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:43:37,921: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:43:37,923: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:43:37,924: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:44:25,714: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:44:25,715: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:44:25,715: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:44:25,715: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:44:25,715: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:44:25,715: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:44:25,729: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:44:25,731: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:44:25,736: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:44:25,737: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:44:25,739: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:44:25,740: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:45:10,661: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:45:10,662: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:45:10,662: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:45:10,662: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:45:10,662: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:45:10,662: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:45:10,684: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:45:10,686: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:45:10,687: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:45:10,689: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:45:10,690: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:45:10,692: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:45:29,520: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:45:29,520: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:45:29,521: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:45:29,521: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:45:29,521: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:45:29,521: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:45:29,544: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:45:29,545: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:45:29,546: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:45:29,547: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:45:29,550: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:45:29,551: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:45:43,723: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:45:43,723: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:45:43,723: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:45:43,723: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:45:43,723: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:45:43,723: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:45:43,748: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:45:43,750: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:45:43,752: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:45:43,754: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:45:43,755: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:45:43,756: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:46:43,541: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:46:43,542: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:46:43,542: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:46:43,542: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:46:43,542: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:46:43,542: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:46:43,565: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:46:43,567: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:46:43,569: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:46:43,570: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:46:43,572: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:46:43,575: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:46:54,687: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:46:54,687: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:46:54,687: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:46:54,687: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:46:54,687: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:46:54,687: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:46:54,712: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:46:54,713: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:46:54,714: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:46:54,716: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:46:54,717: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:46:54,719: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:47:30,872: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:47:30,872: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:47:30,872: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:47:30,872: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:47:30,873: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:47:30,873: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:47:56,975: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:47:56,975: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:47:56,975: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:47:56,975: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:47:56,975: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:47:56,975: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:47:57,005: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:47:57,007: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:47:57,011: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:47:57,013: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:47:57,014: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:47:57,017: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:53:07,415: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:53:07,415: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:53:07,415: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:53:07,415: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:53:07,416: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:53:07,416: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:58:33,952: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:58:33,952: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:58:33,953: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:58:33,953: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:58:33,953: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:58:33,953: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:58:33,989: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:58:33,990: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:58:33,992: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:58:33,995: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:58:33,997: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:58:34,001: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:59:01,371: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:59:01,372: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:59:01,372: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:59:01,373: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:59:01,373: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 03:59:01,373: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:59:01,398: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 03:59:01,399: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 03:59:01,401: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 03:59:01,404: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 03:59:01,405: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 03:59:01,407: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:03:08,700: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:03:08,700: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:03:08,700: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:03:08,701: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:03:08,701: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:03:08,701: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:03:08,723: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:03:08,724: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:03:08,726: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:03:08,728: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:03:08,729: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:03:08,731: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:04:04,565: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:04:04,566: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:04:04,566: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:04:04,566: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:04:04,566: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:04:04,566: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:04:04,590: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:04:04,591: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:04:04,592: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:04:04,594: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:04:04,598: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:04:04,600: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:06:46,248: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:06:46,249: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:06:46,249: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:06:46,249: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:06:46,250: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:06:46,250: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:06:46,284: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:06:46,286: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:06:46,287: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:06:46,289: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:06:46,291: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:06:46,293: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:07:15,769: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:07:15,770: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:07:15,770: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:07:15,770: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:07:15,771: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:07:15,771: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:07:15,806: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:07:15,808: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:07:15,810: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:07:15,811: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:07:15,812: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:07:15,816: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:09:02,361: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:09:02,362: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:09:02,362: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:09:02,362: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:09:02,363: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:09:02,363: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:09:02,391: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:09:02,392: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:09:02,393: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:09:02,395: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:09:02,397: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:09:02,398: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:11:35,955: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:11:35,955: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:11:35,955: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:11:35,955: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:11:35,955: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:11:35,955: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:11:35,983: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:11:35,984: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:11:35,986: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:11:35,987: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:11:35,989: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:11:35,992: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:54:34,391: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:54:34,392: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:54:34,392: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:54:34,392: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:54:34,392: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:54:34,393: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:54:52,997: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:54:52,997: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:54:52,997: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:54:52,997: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:54:52,997: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:54:52,998: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:54:54,754: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:54:54,754: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:54:54,755: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:54:54,755: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:54:54,755: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:54:54,755: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:55:14,439: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:55:14,440: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:55:14,440: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:55:14,440: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:55:14,440: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:55:14,440: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:56:22,903: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:56:22,903: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:56:22,904: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:56:22,904: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:56:22,905: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:56:22,906: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:56:22,929: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:56:22,930: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:56:22,931: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:56:22,932: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:56:22,933: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:56:22,934: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:57:20,822: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:57:20,822: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:57:20,822: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 04:57:20,822: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:57:20,822: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:57:20,822: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:57:20,854: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.germline.png
2018-06-07 04:57:20,855: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.tumour.png
2018-06-07 04:57:20,857: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASPCF.png
2018-06-07 04:57:20,859: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.rawprofile.png
2018-06-07 04:57:20,860: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.sunrise.png
2018-06-07 04:57:20,861: /Users/ryan/PycharmProjects/untitled/tmp/soyirJEYje/somatic/ascatngs/chenmeiyun_T-VS-chenmeiyun_N/chenmeiyun_T.ASCATprofile.png
2018-06-07 05:47:32,847: Not Found: /undefined
2018-06-07 06:58:46,299: Not Found: /undefined
2018-06-07 06:58:49,337: Not Found: /undefined
2018-06-07 06:58:50,489: Not Found: /undefined
2018-06-07 07:15:27,896: {'taskid': 'ncu9h2i5Vg', 'info': '', 'cmd': '实验暂停'}
2018-06-07 08:58:19,821: Not Found: /undefined
2018-06-07 08:59:13,342: Not Found: /undefined
2018-06-12 03:33:54,787: Not Found: /undefined
2018-06-12 03:35:02,223: Internal Server Error: /AnaTaskHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 609, in view
    task_list = json.loads(Task.objects().all().to_json(ensure_ascii=False))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1154, in to_json
    return json_util.dumps(self.as_pymongo(), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 408, in dumps
    return json.dumps(_json_convert(obj, json_options), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in _json_convert
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in <genexpr>
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 99, in _iter_results
    self._populate_cache()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 118, in _populate_cache
    self._result_cache.append(next(self))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1466, in __next__
    raw_doc = next(self._cursor)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1169, in next
    if len(self.__data) or self._refresh():
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1068, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1394, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1372, in start_session
    server_session = self._get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1380, in _get_server_session
    return self._topology.get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 61] Connection refused
2018-06-12 03:45:53,811: Not Found: /undefined
2018-06-12 03:46:27,992: Internal Server Error: /LabTaskHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 550, in view
    task_list = json.loads(Task.objects().all().to_json(ensure_ascii=False))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1154, in to_json
    return json_util.dumps(self.as_pymongo(), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 408, in dumps
    return json.dumps(_json_convert(obj, json_options), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in _json_convert
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in <genexpr>
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 99, in _iter_results
    self._populate_cache()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 118, in _populate_cache
    self._result_cache.append(next(self))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1466, in __next__
    raw_doc = next(self._cursor)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1169, in next
    if len(self.__data) or self._refresh():
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1068, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1394, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1372, in start_session
    server_session = self._get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1380, in _get_server_session
    return self._topology.get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 61] Connection refused
2018-06-12 03:46:34,604: Internal Server Error: /SampleHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1345, in view
    sample_list = json.loads(Sample.objects().all().to_json(ensure_ascii=False))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1154, in to_json
    return json_util.dumps(self.as_pymongo(), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 408, in dumps
    return json.dumps(_json_convert(obj, json_options), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in _json_convert
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in <genexpr>
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 99, in _iter_results
    self._populate_cache()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 118, in _populate_cache
    self._result_cache.append(next(self))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1466, in __next__
    raw_doc = next(self._cursor)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1169, in next
    if len(self.__data) or self._refresh():
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1068, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1394, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1372, in start_session
    server_session = self._get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1380, in _get_server_session
    return self._topology.get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 61] Connection refused
2018-06-12 03:46:37,877: Not Found: /undefined
2018-06-12 03:46:41,196: Internal Server Error: /AnaTaskHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 609, in view
    task_list = json.loads(Task.objects().all().to_json(ensure_ascii=False))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1154, in to_json
    return json_util.dumps(self.as_pymongo(), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 408, in dumps
    return json.dumps(_json_convert(obj, json_options), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in _json_convert
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in <genexpr>
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 99, in _iter_results
    self._populate_cache()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 118, in _populate_cache
    self._result_cache.append(next(self))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1466, in __next__
    raw_doc = next(self._cursor)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1169, in next
    if len(self.__data) or self._refresh():
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1068, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1394, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1372, in start_session
    server_session = self._get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1380, in _get_server_session
    return self._topology.get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 61] Connection refused
2018-06-12 03:47:13,253: Internal Server Error: /LabTaskHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 550, in view
    task_list = json.loads(Task.objects().all().to_json(ensure_ascii=False))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1154, in to_json
    return json_util.dumps(self.as_pymongo(), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 408, in dumps
    return json.dumps(_json_convert(obj, json_options), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in _json_convert
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in <genexpr>
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 99, in _iter_results
    self._populate_cache()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 118, in _populate_cache
    self._result_cache.append(next(self))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1466, in __next__
    raw_doc = next(self._cursor)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1169, in next
    if len(self.__data) or self._refresh():
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1068, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1394, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1372, in start_session
    server_session = self._get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1380, in _get_server_session
    return self._topology.get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 61] Connection refused
2018-06-12 03:47:17,332: Internal Server Error: /SampleHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1345, in view
    sample_list = json.loads(Sample.objects().all().to_json(ensure_ascii=False))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1154, in to_json
    return json_util.dumps(self.as_pymongo(), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 408, in dumps
    return json.dumps(_json_convert(obj, json_options), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in _json_convert
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in <genexpr>
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 99, in _iter_results
    self._populate_cache()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 118, in _populate_cache
    self._result_cache.append(next(self))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1466, in __next__
    raw_doc = next(self._cursor)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1169, in next
    if len(self.__data) or self._refresh():
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1068, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1394, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1372, in start_session
    server_session = self._get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1380, in _get_server_session
    return self._topology.get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 61] Connection refused
2018-06-12 03:58:27,953: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 471, in allocate
    data['analyst']=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 03:58:51,341: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 465, in allocate
    data = json.loads(request.body.decode('utf-8'))
  File "/Users/ryan/anaconda3/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/Users/ryan/anaconda3/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Users/ryan/anaconda3/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2018-06-12 03:58:58,195: Internal Server Error: /PMTaskHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 398, in view
    task_list = json.loads(Task.objects().all().to_json(ensure_ascii=False))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1154, in to_json
    return json_util.dumps(self.as_pymongo(), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 408, in dumps
    return json.dumps(_json_convert(obj, json_options), *args, **kwargs)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in _json_convert
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/bson/json_util.py", line 449, in <genexpr>
    return list((_json_convert(v, json_options) for v in obj))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 99, in _iter_results
    self._populate_cache()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/queryset.py", line 118, in _populate_cache
    self._result_cache.append(next(self))
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/queryset/base.py", line 1466, in __next__
    raw_doc = next(self._cursor)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1169, in next
    if len(self.__data) or self._refresh():
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/cursor.py", line 1068, in _refresh
    self.__session = self.__collection.database.client._ensure_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1394, in _ensure_session
    return self.start_session(causal_consistency=False)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1372, in start_session
    server_session = self._get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/mongo_client.py", line 1380, in _get_server_session
    return self._topology.get_server_session()
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 427, in get_server_session
    None)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/pymongo/topology.py", line 199, in _select_servers_loop
    self._error_message(selector))
pymongo.errors.ServerSelectionTimeoutError: localhost:27017: [Errno 61] Connection refused
2018-06-12 04:00:30,087: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 465, in allocate
    data = json.loads(request.body.decode('utf-8'))
  File "/Users/ryan/anaconda3/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/Users/ryan/anaconda3/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Users/ryan/anaconda3/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2018-06-12 04:00:41,779: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 471, in allocate
    data['analyst']=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 04:01:23,380: Internal Server Error: /AnaTaskHandle/qcview/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 641, in qcview
    workdir = os.path.join(self.workdir, task.pk)
AttributeError: 'NoneType' object has no attribute 'pk'
2018-06-12 04:01:40,073: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 471, in allocate
    data['analyst']=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 04:01:43,812: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 471, in allocate
    data['analyst']=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 04:02:00,966: Internal Server Error: /ProjectHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1054, in view
    task = json.loads(Task.objects(pk=task).first().to_json(ensure_ascii=False))
AttributeError: 'NoneType' object has no attribute 'to_json'
2018-06-12 04:02:08,512: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 471, in allocate
    data['analyst']=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 04:02:10,191: Internal Server Error: /ProjectHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1054, in view
    task = json.loads(Task.objects(pk=task).first().to_json(ensure_ascii=False))
AttributeError: 'NoneType' object has no attribute 'to_json'
2018-06-12 04:02:15,760: Internal Server Error: /ProjectHandle/init/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 908, in init
    user_list=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 04:02:17,537: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 471, in allocate
    data['analyst']=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 12:03:43,939: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 471, in allocate
    data['analyst']=self.ldapc.getGroupUsers('Analyst')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0]['memberUid']
TypeError: tuple indices must be integers or slices, not str
2018-06-12 12:06:09,138: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 472, in allocate
    data['parser']=self.ldapc.getGroupUsers('parser')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0][1]['memberUid']
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 852, in search_s
    return self.search_ext_s(base,scope,filterstr,attrlist,attrsonly,None,None,timeout=self.timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 846, in search_ext_s
    return self.result(msgid,all=1,timeout=timeout)[1]
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 738, in result
    resp_type, resp_data, resp_msgid = self.result2(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 742, in result2
    resp_type, resp_data, resp_msgid, resp_ctrls = self.result3(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 749, in result3
    resp_ctrl_classes=resp_ctrl_classes
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 756, in result4
    ldap_result = self._ldap_call(self._l.result4,msgid,all,timeout,add_ctrls,add_intermediates,add_extop)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 329, in _ldap_call
    reraise(exc_type, exc_value, exc_traceback)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/compat.py", line 44, in reraise
    raise exc_value
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 313, in _ldap_call
    result = func(*args,**kwargs)
ldap.NO_SUCH_OBJECT: {'desc': 'No such object', 'matched': 'ou=Roles,dc=nodomain'}
2018-06-12 12:06:48,152: Internal Server Error: /
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 91, in index
    return render('index.html')
TypeError: render() missing 1 required positional argument: 'template_name'
2018-06-12 12:06:48,500: Not Found: /favicon.ico
2018-06-12 12:07:00,657: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 472, in allocate
    data['parser']=self.ldapc.getGroupUsers('parser')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0][1]['memberUid']
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 852, in search_s
    return self.search_ext_s(base,scope,filterstr,attrlist,attrsonly,None,None,timeout=self.timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 846, in search_ext_s
    return self.result(msgid,all=1,timeout=timeout)[1]
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 738, in result
    resp_type, resp_data, resp_msgid = self.result2(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 742, in result2
    resp_type, resp_data, resp_msgid, resp_ctrls = self.result3(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 749, in result3
    resp_ctrl_classes=resp_ctrl_classes
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 756, in result4
    ldap_result = self._ldap_call(self._l.result4,msgid,all,timeout,add_ctrls,add_intermediates,add_extop)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 329, in _ldap_call
    reraise(exc_type, exc_value, exc_traceback)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/compat.py", line 44, in reraise
    raise exc_value
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 313, in _ldap_call
    result = func(*args,**kwargs)
ldap.NO_SUCH_OBJECT: {'desc': 'No such object', 'matched': 'ou=Roles,dc=nodomain'}
2018-06-12 12:07:49,135: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 472, in allocate
    data['parser']=self.ldapc.getGroupUsers('parser')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0][1]['memberUid']
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 852, in search_s
    return self.search_ext_s(base,scope,filterstr,attrlist,attrsonly,None,None,timeout=self.timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 846, in search_ext_s
    return self.result(msgid,all=1,timeout=timeout)[1]
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 738, in result
    resp_type, resp_data, resp_msgid = self.result2(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 742, in result2
    resp_type, resp_data, resp_msgid, resp_ctrls = self.result3(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 749, in result3
    resp_ctrl_classes=resp_ctrl_classes
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 756, in result4
    ldap_result = self._ldap_call(self._l.result4,msgid,all,timeout,add_ctrls,add_intermediates,add_extop)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 329, in _ldap_call
    reraise(exc_type, exc_value, exc_traceback)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/compat.py", line 44, in reraise
    raise exc_value
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 313, in _ldap_call
    result = func(*args,**kwargs)
ldap.NO_SUCH_OBJECT: {'desc': 'No such object', 'matched': 'ou=Roles,dc=nodomain'}
2018-06-12 12:10:39,975: Internal Server Error: /ProjectHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1054, in view
    task = json.loads(Task.objects(pk=task).first().to_json(ensure_ascii=False))
AttributeError: 'NoneType' object has no attribute 'to_json'
2018-06-12 12:10:42,885: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 472, in allocate
    data['parser']=self.ldapc.getGroupUsers('parser')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0][1]['memberUid']
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 852, in search_s
    return self.search_ext_s(base,scope,filterstr,attrlist,attrsonly,None,None,timeout=self.timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 846, in search_ext_s
    return self.result(msgid,all=1,timeout=timeout)[1]
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 738, in result
    resp_type, resp_data, resp_msgid = self.result2(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 742, in result2
    resp_type, resp_data, resp_msgid, resp_ctrls = self.result3(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 749, in result3
    resp_ctrl_classes=resp_ctrl_classes
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 756, in result4
    ldap_result = self._ldap_call(self._l.result4,msgid,all,timeout,add_ctrls,add_intermediates,add_extop)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 329, in _ldap_call
    reraise(exc_type, exc_value, exc_traceback)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/compat.py", line 44, in reraise
    raise exc_value
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 313, in _ldap_call
    result = func(*args,**kwargs)
ldap.NO_SUCH_OBJECT: {'desc': 'No such object', 'matched': 'ou=Roles,dc=nodomain'}
2018-06-12 12:10:45,646: Internal Server Error: /ProjectHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1054, in view
    task = json.loads(Task.objects(pk=task).first().to_json(ensure_ascii=False))
AttributeError: 'NoneType' object has no attribute 'to_json'
2018-06-12 12:13:05,240: Internal Server Error: /ProjectHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1054, in view
    task = json.loads(Task.objects(pk=task).first().to_json(ensure_ascii=False))
AttributeError: 'NoneType' object has no attribute 'to_json'
2018-06-12 12:13:09,560: Internal Server Error: /PMTaskHandle/allocate/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 472, in allocate
    data['parser']=self.ldapc.getGroupUsers('parser')
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 103, in getGroupUsers
    rawdata=self.con.search_s('cn=%s,ou=Roles,DC=nodomain'%group,ldap.SCOPE_SUBTREE,attrlist=['memberUid'])[0][1]['memberUid']
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 852, in search_s
    return self.search_ext_s(base,scope,filterstr,attrlist,attrsonly,None,None,timeout=self.timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 846, in search_ext_s
    return self.result(msgid,all=1,timeout=timeout)[1]
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 738, in result
    resp_type, resp_data, resp_msgid = self.result2(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 742, in result2
    resp_type, resp_data, resp_msgid, resp_ctrls = self.result3(msgid,all,timeout)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 749, in result3
    resp_ctrl_classes=resp_ctrl_classes
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 756, in result4
    ldap_result = self._ldap_call(self._l.result4,msgid,all,timeout,add_ctrls,add_intermediates,add_extop)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 329, in _ldap_call
    reraise(exc_type, exc_value, exc_traceback)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/compat.py", line 44, in reraise
    raise exc_value
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/ldap/ldapobject.py", line 313, in _ldap_call
    result = func(*args,**kwargs)
ldap.NO_SUCH_OBJECT: {'desc': 'No such object', 'matched': 'ou=Roles,dc=nodomain'}
2018-06-12 12:14:38,545: Internal Server Error: /ProjectHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1054, in view
    task = json.loads(Task.objects(pk=task).first().to_json(ensure_ascii=False))
AttributeError: 'NoneType' object has no attribute 'to_json'
2018-06-12 13:09:19,661: Internal Server Error: /ProjectHandle/view/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 1057, in view
    task = json.loads(Task.objects(pk=task).first().to_json(ensure_ascii=False))
AttributeError: 'NoneType' object has no attribute 'to_json'
2018-06-12 13:11:10,065: add task: cCi4uktJhe 0000001 MT100000 肺癌
2018-06-12 13:11:10,079: add task: 0ECGRLwbfs 0000001 MT100000 胃癌
2018-06-12 13:11:10,083: [<Task: cCi4uktJhe>, <Task: 0ECGRLwbfs>]
2018-06-12 13:11:10,106: add task: eJ2lZs4cka 0000001 MT100000 
2018-06-12 13:11:10,109: Cannot encode object: <Task: eJ2lZs4cka>
2018-06-12 13:11:10,111: add task: cyZKRJEIns 0000001 MT100000 
2018-06-12 13:11:10,115: Cannot encode object: <Task: cyZKRJEIns>
2018-06-12 13:11:10,122: cmd: {"taskid": "cCi4uktJhe", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0", "cmd": "\u6682\u505c"}
2018-06-12 13:11:10,131: cmd: {"taskid": "cCi4uktJhe", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0"}
2018-06-12 13:11:10,133: Internal Server Error: /LabTaskHandle/cmd/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 493, in cmd
    cmd=data.pop('cmd')
KeyError: 'cmd'
2018-06-12 13:11:10,203: cmd: {"taskid": "cCi4uktJhe", "cmd": "\u7ec8\u6b62", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0"}
2018-06-12 13:11:10,227: {'taskid': 'cCi4uktJhe', 'samples': [{'sampleid': '18A00335XJ03', 'tumortype': '肺癌', 'tissue': '血浆', 'read1': 'TACACTCTTTCCCTACACGACGCTCTTCCGATCT', 'read2': 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC', 'platform': 'Nextseq', 'i5': 'MID', 'i7': 'I4', 'volume': 5, 'concentration': '', 'QsepPeak': '', 'qPCRConcentration': '', 'datasize': 5, 'info': ''}, {'sampleid': '18A00000XY122', 'tumortype': '肺癌', 'tissue': '血液', 'read1': 'TACACTCTTTCCCTACACGACGCTCTTCCGATCT', 'read2': 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC', 'platform': 'Nextseq', 'i5': 'MID', 'i7': 'I4', 'volume': 5, 'concentration': '', 'QsepPeak': '', 'qPCRConcentration': '', 'datasize': 5, 'info': ''}]}
2018-06-12 13:11:10,320: Internal Server Error: /ProjectHandle/init/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 884, in init
    project = Project(**data)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/base/document.py", line 77, in __init__
    raise FieldDoesNotExist(msg)
mongoengine.errors.FieldDoesNotExist: The fields "{'info'}" do not exist on the document "Project"
2018-06-12 13:11:32,586: add task: NZsaVD0iAb 0000001 MT100000 肺癌
2018-06-12 13:11:32,598: add task: pSHVYQfCjW 0000001 MT100000 胃癌
2018-06-12 13:11:32,611: [<Task: NZsaVD0iAb>, <Task: pSHVYQfCjW>]
2018-06-12 13:11:32,654: add task: 4Os139HkPG 0000001 MT100000 
2018-06-12 13:11:32,658: Cannot encode object: <Task: 4Os139HkPG>
2018-06-12 13:11:32,670: add task: 1zlJIWUVFN 0000001 MT100000 
2018-06-12 13:11:32,682: Cannot encode object: <Task: 1zlJIWUVFN>
2018-06-12 13:11:32,727: cmd: {"taskid": "NZsaVD0iAb", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0", "cmd": "\u6682\u505c"}
2018-06-12 13:11:32,766: cmd: {"taskid": "NZsaVD0iAb", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0"}
2018-06-12 13:11:32,774: Internal Server Error: /LabTaskHandle/cmd/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 493, in cmd
    cmd=data.pop('cmd')
KeyError: 'cmd'
2018-06-12 13:11:32,826: cmd: {"taskid": "NZsaVD0iAb", "cmd": "\u7ec8\u6b62", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0"}
2018-06-12 13:11:32,841: {'taskid': 'NZsaVD0iAb', 'samples': [{'sampleid': '18A00335XJ03', 'tumortype': '肺癌', 'tissue': '血浆', 'read1': 'TACACTCTTTCCCTACACGACGCTCTTCCGATCT', 'read2': 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC', 'platform': 'Nextseq', 'i5': 'MID', 'i7': 'I4', 'volume': 5, 'concentration': '', 'QsepPeak': '', 'qPCRConcentration': '', 'datasize': 5, 'info': ''}, {'sampleid': '18A00000XY122', 'tumortype': '肺癌', 'tissue': '血液', 'read1': 'TACACTCTTTCCCTACACGACGCTCTTCCGATCT', 'read2': 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC', 'platform': 'Nextseq', 'i5': 'MID', 'i7': 'I4', 'volume': 5, 'concentration': '', 'QsepPeak': '', 'qPCRConcentration': '', 'datasize': 5, 'info': ''}]}
2018-06-12 13:11:32,938: Internal Server Error: /ProjectHandle/init/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 884, in init
    project = Project(**data)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/base/document.py", line 77, in __init__
    raise FieldDoesNotExist(msg)
mongoengine.errors.FieldDoesNotExist: The fields "{'info'}" do not exist on the document "Project"
2018-06-12 13:12:13,441: add task: TdBgsP0Rvr 0000001 MT100000 肺癌
2018-06-12 13:12:13,467: add task: YibUa4Zt7m 0000001 MT100000 胃癌
2018-06-12 13:12:13,474: [<Task: TdBgsP0Rvr>, <Task: YibUa4Zt7m>]
2018-06-12 13:12:13,517: add task: 3wuVn1G9lB 0000001 MT100000 
2018-06-12 13:12:13,520: Cannot encode object: <Task: 3wuVn1G9lB>
2018-06-12 13:12:13,523: add task: AdY8mDlpBi 0000001 MT100000 
2018-06-12 13:12:13,526: Cannot encode object: <Task: AdY8mDlpBi>
2018-06-12 13:12:13,531: cmd: {"taskid": "TdBgsP0Rvr", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0", "cmd": "\u6682\u505c"}
2018-06-12 13:12:13,548: cmd: {"taskid": "TdBgsP0Rvr", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0"}
2018-06-12 13:12:13,552: Internal Server Error: /LabTaskHandle/cmd/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 493, in cmd
    cmd=data.pop('cmd')
KeyError: 'cmd'
2018-06-12 13:12:13,583: cmd: {"taskid": "TdBgsP0Rvr", "cmd": "\u7ec8\u6b62", "info": "\u968f\u4fbf\u586b\u5199\u539f\u56e0"}
2018-06-12 13:12:13,608: {'taskid': 'TdBgsP0Rvr', 'samples': [{'sampleid': '18A00335XJ03', 'tumortype': '肺癌', 'tissue': '血浆', 'read1': 'TACACTCTTTCCCTACACGACGCTCTTCCGATCT', 'read2': 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC', 'platform': 'Nextseq', 'i5': 'MID', 'i7': 'I4', 'volume': 5, 'concentration': '', 'QsepPeak': '', 'qPCRConcentration': '', 'datasize': 5, 'info': ''}, {'sampleid': '18A00000XY122', 'tumortype': '肺癌', 'tissue': '血液', 'read1': 'TACACTCTTTCCCTACACGACGCTCTTCCGATCT', 'read2': 'AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC', 'platform': 'Nextseq', 'i5': 'MID', 'i7': 'I4', 'volume': 5, 'concentration': '', 'QsepPeak': '', 'qPCRConcentration': '', 'datasize': 5, 'info': ''}]}
2018-06-12 13:12:13,704: Internal Server Error: /ProjectHandle/init/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 884, in init
    project = Project(**data)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/mongoengine/base/document.py", line 77, in __init__
    raise FieldDoesNotExist(msg)
mongoengine.errors.FieldDoesNotExist: The fields "{'info'}" do not exist on the document "Project"
2018-06-12 13:12:13,736: Internal Server Error: /AnaTaskHandle/qcview/
Traceback (most recent call last):
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/ryan/PycharmProjects/untitled/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/ryan/PycharmProjects/untitled/projectmanage/views.py", line 644, in qcview
    workdir = os.path.join(self.workdir, task.pk)
AttributeError: 'NoneType' object has no attribute 'pk'
2018-06-12 13:34:23,492: Not Found: /undefined
2018-06-12 13:34:27,124: Not Found: /undefined
2018-06-12 13:34:44,455: Not Found: /undefined
2018-06-12 13:34:54,456: Not Found: /undefined
2018-06-12 13:41:48,087: Not Found: /undefined
2018-06-12 13:47:38,352: Not Found: /undefined
2018-06-12 13:47:40,918: Not Found: /undefined
2018-06-12 13:54:47,060: Not Found: /undefined
2018-06-12 13:59:31,945: Not Found: /undefined
